### PR TITLE
Fix documentation typo

### PR DESCRIPTION
**Pull request by 2shortplanks**
Simple documentation typo fix

### CODE REVIEW

File: lib/builtin.pm [hunk 1]
```diff
@@ -134,7 +134,7 @@ bundles.
 
     $val = false;
 
-Returns the boolean fiction value. While any non-true scalar value is
+Returns the boolean false value. While any non-true scalar value is
```
Comment (ilmari) [inline lib/builtin.pm:137]: I think the documentation for `true` above should be changed to match too, i.e. "the boolean true value."

**Comment by johannessen** [comment]
I don’t think this is a typo. It looks like just an artistic choice of words.

**Comment by jkeenan** [comment]
> I don’t think this is a typo. It looks like just an artistic choice of words.

@leonerd, the terminology "boolean fiction value" was introduced in 6a2e756f869.  Would you care to comment on this?

**Comment by khwilliamson** [comment]
Even if it is artistic, if its meaning isn't obvious, it should be changed.  I kinda see what it means, and I'm a native English speaker.

**Comment by johannessen** [comment]
It’s nice for Perl documentation to have some personality.

`true` (“boolean truth”) is right above `false` (“boolean fiction”) in [builtin](https://perldoc.perl.org/5.40.0/builtin#FUNCTIONS). I personally think reading it in context like that makes the meaning very obvious. But I already know the meaning, so I’m ***not*** the best person to judge that. And if it’s not obvious, it should indeed be changed.

**Review by jkeenan — APPROVED** [review]

**Comment by jkeenan** [comment]
> Even if it is artistic, if its meaning isn't obvious, it should be changed. I kinda see what it means, and I'm a native English speaker.

I agree.  Clarity is more important than aesthetics.  I'm merging @2shortplanks's patch.  @ilmari I suggest creating a separate p.r. for `true`.

**Pull request merged**